package.json

```diff
@@ -7,7 +7,7 @@
     "start:example:js": "yarn build && npx http-server ./packages/js -o /example",
     "lint": "eslint . --ext .ts,.tsx",
     "test": "jest",
-    "prettier": "prettier --write './packages/**/**/*.{ts,tsx}'",
+    "prettier": "prettier --write ./packages/**/src/*.{ts,tsx}",
```

Comment:
I think we want to target all the ts / tsx files, no need to specify src imho

Reply:
That would mean that we'd run prettier also on compiled files (`/lib` and `/es`), is that correct?

Reply:
no definitely not, we should put these directories in a `.prettierignore` file. In this case I think we should rather blacklist instead of whitelist directories

Reply:
Or done in ec0fff21dca11b67d22799d229a07b1ac3371c08